src/DogsInfo.tsx

```diff
@@ -0,0 +1,46 @@
+import { DogInfo } from "./DogInfoCard";
+
+export const dogsInfo: DogInfo[] = [
```

Comment:
nit: use all caps for constant
```suggestion
export const DOGS_INFO: DogInfo[] = [
```